bundles/org.openhab.voice.mimictts/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -6,6 +6,7 @@
 	<type>voice</type>
 	<name>macOS Text-to-Speech</name>
 	<description>The macOS Text-to-Speech (TTS) service uses the macOS "say" command for producing spoken text.</description>
```

Comment:
I just saw this copy/paste error, WDYT about this @dalgwen?

```suggestion
	<name>Mimic Text-to-Speech</name>
	<description>An offline open source Text-To-speech engine designed by Mycroft A.I.</description>
```

Reply:
I decided to merge and not let this PR be blocked by this unrelated finding. Please create another PR to fix this when agreed upon.

Reply:
I've created https://github.com/openhab/openhab-addons/pull/14821 for this.

Reply:
Ooops, sorry ! Thanks for the fix.